### ISSUE TITLE
'time' is a cell line

### ISSUE THREAD

**Issue by danebell** [clulab/reach#443]
> At the same time , thecatalytic activity of KSR1 was confirmed by the fact that tumour necrosis factor alpha ( TNFalpha ) and ceramide can induce KSR1 autophosphorylation and increase its capacity to phosphorylate and activate Raf-1

`time` is recognized as a cellosaurus cell line. This and any other cell lines that are more likely to be common English words should be filtered out during NER.

**Comment by hickst** [comment]
Yea, biologists seem to love these kind of computer-vexing names. We can add 'time' but the general problem is finding the "common" English words among our 28 KBs which total 4M+ entries. If you've got a list of (say a couple hundred) common words, I can check those against our KBs and add any conflicts to the stop list. On the downside, a couple hundred is a small set (unless you're really good at out-guessing the biologists).

**Comment by MihaiSurdeanu** [comment]
For now we can incrementally remove these by adding them to the
ner_stoplist file:
https://github.com/clulab/bioresources/blob/master/src/main/resources/org/clulab/reach/kb/ner_stoplist.txt

**Comment by hickst** [comment]
Yes and that's what I am doing in this case, of course. I was just suggesting that, if we could identify more potential candidates, I could test them against the KBs for conflicts and add the conflicts to the stop list. Currently the stop list is only 33 words (now 34).

**Comment by myedibleenso** [comment]
We have a set of w2v embeddings trained on gigaword on the file server.  I grabbed those, filtered out the few entries that didn't start with a-z, and then took their intersection with the terms listed in `/usr/share/dict/words`.  I don't have frequency information on these words and I know the list is a bit noisy (ex. acronyms, single letter entries), but I am curious to see how many of these terms overlap with KB entries.  Perhaps we can just look at what case-folded, single-token entries in our KBs exist in this set of words?  Hopefully that is not such a large list...

[vocab.txt](https://github.com/clulab/reach/files/579853/vocab.txt)

**Comment by MihaiSurdeanu** [comment]
Very nice!

**Comment by hickst** [comment]
Working on getting you that key list now.

**Comment by hickst** [comment]
I got the key list down to ~186k by dropping the PubChem KB and filtering entries to single "words". The file is on Jenny: /net/kate/storage/work/hickst/temp/wordKEYS

**Comment by hickst** [comment]
GHP said:

> I just checked: it looks like 3K terms overlap. many of these are valid

**Comment by MihaiSurdeanu** [comment]
Let's discuss today.

**Comment by myedibleenso** [comment]
@hickst, can you please generate another KB dump that doesn't include uniprot and the organs kb?

**Comment by hickst** [comment]
@myedibleenso the new one, without Uniprot proteins and the Uberon organ KB (and w/o any chemicals) is in: /net/kate/storage/work/hickst/temp/wordKEYS2
